proto/contractmanager/query.proto

```diff
@@ -0,0 +1,51 @@
+syntax = "proto3";
+package neutronorg.neutron.contractmanager;
+
+import "gogoproto/gogo.proto";
+import "google/api/annotations.proto";
+import "cosmos/base/query/v1beta1/pagination.proto";
+import "contractmanager/params.proto";
+import "contractmanager/genesis.proto";
+// this line is used by starport scaffolding # 1
+
+option go_package = "github.com/neutron-org/neutron/x/contractmanager/types";
+
+// Query defines the gRPC querier service.
+service Query {
+	// Parameters queries the parameters of the module.
+	rpc Params(QueryParamsRequest) returns (QueryParamsResponse) {
+		option (google.api.http).get = "/neutron-org/neutron/contractmanager/params";
+	}
+	// Queries a Failure by address.
+	rpc Failures(QueryFailuresRequest) returns (QueryFailuresResponse) {
+		option (google.api.http).get = "/neutron-org/neutron/contractmanager/failure/{address}";
+	}
+
+	// Queries a list of Failure items.
+	rpc AllFailures(QueryFailuresRequest) returns (QueryFailuresResponse) {
```

Comment:
i do not see any difference between `AllFailures` and `Failures`. And code is the same for both methods. Method named `AllFailures` should at least omit the filtering with address params

Reply:
the difference in the `address` parameter. But we need to have two grpc paths, one with address parameter another without. I am not sure that if I remove it (I mean second `rpc` endpoint) it will work properly with and without `address` parameter.

Reply:
I got the idea, i mean there is no difference which method to call via g(rpc), both takes the same arguments(paging, address), both executes the same code. What do you think to swap names `func (k Keeper) Failures` and `func (k Keeper) AllFailures`, its strange, when the more generic method `Failures` calls the more specific `AllFailures` and passes the filter arguments.

I've removed `AllFailures` from proto file, regenerated the code, and `Failures` works properly with empty `address`, but trailing slash is required.

Reply:
Tested. If I will remove one of the endpoints then other will stop working. It means if I will remove `AllFailures` rpc, then it will be longer not possible to get list of all failures. Same with other, in this case it will not be possible to get `Failures` for specific address.